lab-kyle/data/pokemon/0dfbbbaa-a7c3-441a-acea-a3aefa94f4cf.json

```diff
@@ -0,0 +1 @@
+{"id":"0dfbbbaa-a7c3-441a-acea-a3aefa94f4cf","name":"greninja","color":"blue"}
```

Comment:
You're probably okay to gitignore that data directory. I don't think we need the stored files on GH?